File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Xrts/XRT0201U00GrdJaeryoItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Xrts
{
    [Serializable]
	public class XRT0201U00GrdJaeryoItemInfo
	{
		private String _jaeryoCode;
		private String _jaeryoName;
		private String _suryang;
		private String _ordDanui;
		private String _pkinv1001;
		private String _bunho;
		private String _orderDate;
		private String _inOutGubun;
		private String _actingDate;
		private String _actingBuseo;
		private String _fkocsInv;
		private String _fkocsXrt;
		private String _danuiName;
		private String _bunryu2;
		private String _jaeryoGubun;
		private String _jaeryoYn;
		private String _inputControl;
		private String _bunCode;
		private String _nalsu;
		private String _dataRowState;

		public String JaeryoCode
		{
			get { return this._jaeryoCode; }
			set { this._jaeryoCode = value; }
		}

		public String JaeryoName
		{
			get { return this._jaeryoName; }
			set { this._jaeryoName = value; }
		}

		public String Suryang
		{
			get { return this._suryang; }
			set { this._suryang = value; }
		}

		public String OrdDanui
		{
			get { return this._ordDanui; }
			set { this._ordDanui = value; }
		}

		public String Pkinv1001
		{
			get { return this._pkinv1001; }
			set { this._pkinv1001 = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String OrderDate
		{
			get { return this._orderDate; }
			set { this._orderDate = value; }
		}

		public String InOutGubun
		{
			get { return this._inOutGubun; }
			set { this._inOutGubun = value; }
		}

		public String ActingDate
		{
			get { return this._actingDate; }
			set { this._actingDate = value; }
		}

		public String ActingBuseo
		{
			get { return this._actingBuseo; }
			set { this._actingBuseo = value; }
		}

		public String FkocsInv
		{
			get { return this._fkocsInv; }
			set { this._fkocsInv = value; }
		}

		public String FkocsXrt
		{
			get { return this._fkocsXrt; }
			set { this._fkocsXrt = value; }
		}

		public String DanuiName
		{
			get { return this._danuiName; }
			set { this._danuiName = value; }
		}

		public String Bunryu2
		{
			get { return this._bunryu2; }
			set { this._bunryu2 = value; }
		}

		public String JaeryoGubun
		{
			get { return this._jaeryoGubun; }
			set { this._jaeryoGubun = value; }
		}

		public String JaeryoYn
		{
			get { return this._jaeryoYn; }
			set { this._jaeryoYn = value; }
		}

		public String InputControl
		{
			get { return this._inputControl; }
			set { this._inputControl = value; }
		}

		public String BunCode
		{
			get { return this._bunCode; }
			set { this._bunCode = value; }
		}

		public String Nalsu
		{
			get { return this._nalsu; }
			set { this._nalsu = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public XRT0201U00GrdJaeryoItemInfo() { }

		public XRT0201U00GrdJaeryoItemInfo(String jaeryoCode, String jaeryoName, String suryang, String ordDanui, String pkinv1001, String bunho, String orderDate, String inOutGubun, String actingDate, String actingBuseo, String fkocsInv, String fkocsXrt, String danuiName, String bunryu2, String jaeryoGubun, String jaeryoYn, String inputControl, String bunCode, String nalsu, String dataRowState)
		{
			this._jaeryoCode = jaeryoCode;
			this._jaeryoName = jaeryoName;
			this._suryang = suryang;
			this._ordDanui = ordDanui;
			this._pkinv1001 = pkinv1001;
			this._bunho = bunho;
			this._orderDate = orderDate;
			this._inOutGubun = inOutGubun;
			this._actingDate = actingDate;
			this._actingBuseo = actingBuseo;
			this._fkocsInv = fkocsInv;
			this._fkocsXrt = fkocsXrt;
			this._danuiName = danuiName;
			this._bunryu2 = bunryu2;
			this._jaeryoGubun = jaeryoGubun;
			this._jaeryoYn = jaeryoYn;
			this._inputControl = inputControl;
			this._bunCode = bunCode;
			this._nalsu = nalsu;
			this._dataRowState = dataRowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/xrts/XRT0201U00GrdJaeryoItemInfo.java
package nta.med.data.model.ihis.xrts;

import java.math.BigDecimal;
import java.util.Date;

public class XRT0201U00GrdJaeryoItemInfo {
	private String jaeryoCode;
	private String jaeryoName;
	private String suryang;
	private String ordDanui;
	private Double pkinv1001;
	private String bunho;
	private String orderDate;
	private String inOutGubun;
	private Date actingDate;
	private String actingBuseo;
	private Double fkocsInv;
	private BigDecimal fkocsXrt;
	private String danuiName;
	private String bunryu2;
	private String jaeryoGubun;
	private String jaeryoYn;
	private String inputControl;
	private String bunCode;
	private Double nalsu;

	public XRT0201U00GrdJaeryoItemInfo(String jaeryoCode, String jaeryoName,
			String suryang, String ordDanui, Double pkinv1001, String bunho,
			String orderDate, String inOutGubun, Date actingDate,
			String actingBuseo, Double fkocsInv, BigDecimal fkocsXrt,
			String danuiName, String bunryu2, String jaeryoGubun,
			String jaeryoYn, String inputControl, String bunCode, Double nalsu) {
		super();
		this.jaeryoCode = jaeryoCode;
		this.jaeryoName = jaeryoName;
		this.suryang = suryang;
		this.ordDanui = ordDanui;
		this.pkinv1001 = pkinv1001;
		this.bunho = bunho;
		this.orderDate = orderDate;
		this.inOutGubun = inOutGubun;
		this.actingDate = actingDate;
		this.actingBuseo = actingBuseo;
		this.fkocsInv = fkocsInv;
		this.fkocsXrt = fkocsXrt;
		this.danuiName = danuiName;
		this.bunryu2 = bunryu2;
		this.jaeryoGubun = jaeryoGubun;
		this.jaeryoYn = jaeryoYn;
		this.inputControl = inputControl;
		this.bunCode = bunCode;
		this.nalsu = nalsu;
	}

	public String getJaeryoCode() {
		return jaeryoCode;
	}

	public void setJaeryoCode(String jaeryoCode) {
		this.jaeryoCode = jaeryoCode;
	}

	public String getJaeryoName() {
		return jaeryoName;
	}

	public void setJaeryoName(String jaeryoName) {
		this.jaeryoName = jaeryoName;
	}

	public String getSuryang() {
		return suryang;
	}

	public void setSuryang(String suryang) {
		this.suryang = suryang;
	}

	public String getOrdDanui() {
		return ordDanui;
	}

	public void setOrdDanui(String ordDanui) {
		this.ordDanui = ordDanui;
	}

	public Double getPkinv1001() {
		return pkinv1001;
	}

	public void setPkinv1001(Double pkinv1001) {
		this.pkinv1001 = pkinv1001;
	}

	public String getBunho() {
		return bunho;
	}

	public void setBunho(String bunho) {
		this.bunho = bunho;
	}

	public String getOrderDate() {
		return orderDate;
	}

	public void setOrderDate(String orderDate) {
		this.orderDate = orderDate;
	}

	public String getInOutGubun() {
		return inOutGubun;
	}

	public void setInOutGubun(String inOutGubun) {
		this.inOutGubun = inOutGubun;
	}

	public Date getActingDate() {
		return actingDate;
	}

	public void setActingDate(Date actingDate) {
		this.actingDate = actingDate;
	}

	public String getActingBuseo() {
		return actingBuseo;
	}

	public void setActingBuseo(String actingBuseo) {
		this.actingBuseo = actingBuseo;
	}

	public Double getFkocsInv() {
		return fkocsInv;
	}

	public void setFkocsInv(Double fkocsInv) {
		this.fkocsInv = fkocsInv;
	}

	public BigDecimal getFkocsXrt() {
		return fkocsXrt;
	}

	public void setFkocsXrt(BigDecimal fkocsXrt) {
		this.fkocsXrt = fkocsXrt;
	}

	public String getDanuiName() {
		return danuiName;
	}

	public void setDanuiName(String danuiName) {
		this.danuiName = danuiName;
	}

	public String getBunryu2() {
		return bunryu2;
	}

	public void setBunryu2(String bunryu2) {
		this.bunryu2 = bunryu2;
	}

	public String getJaeryoGubun() {
		return jaeryoGubun;
	}

	public void setJaeryoGubun(String jaeryoGubun) {
		this.jaeryoGubun = jaeryoGubun;
	}

	public String getJaeryoYn() {
		return jaeryoYn;
	}

	public void setJaeryoYn(String jaeryoYn) {
		this.jaeryoYn = jaeryoYn;
	}

	public String getInputControl() {
		return inputControl;
	}

	public void setInputControl(String inputControl) {
		this.inputControl = inputControl;
	}

	public String getBunCode() {
		return bunCode;
	}

	public void setBunCode(String bunCode) {
		this.bunCode = bunCode;
	}

	public Double getNalsu() {
		return nalsu;
	}

	public void setNalsu(Double nalsu) {
		this.nalsu = nalsu;
	}

}